cmd/mt-whisper-importer-writer/main.go

```diff
@@ -33,6 +33,7 @@ var (
 	uriPath         = flag.String("uri-path", "/chunks", "the URI on which we expect chunks to get posted")
 	numPartitions   = flag.Int("num-partitions", 1, "Number of Partitions")
 	logLevel        = flag.String("log-level", "info", "log level. panic|fatal|error|warning|info|debug")
+	orgId           = flag.Int("org-id", -1, "the id of the org we import to")
```

Comment:
I think we should set the default to '0', which should mean dont do org-id check.